lib/resources/cucumber_json_merge.rb

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env ruby
+#
+# This script can be used to merge JSON files produced by the JSON output format
+# used by Cucumber (or, probably, any JSON files, really).
+# It allows you to split up your run using the parallel_tests gem, produce JSON
+# results for each split, and recombine those result files into something the
+# Cucumber Reports plugin will recognize.
+#
+# Example:
+# parallel_cucumber features/ -o '-p parallel_master_no_nightly'
+# cucumber_json_merge.rb -o ../kuality-kfs-cu-output.json ../kuality-kfs-cu-output.split*.json
```

Comment:
Changing to kuality-kfs as requested.  Doing so has implications to the Jenkins jobs that run on the server for the daily run.  Those server side  change will not be required for a while so creating another user story to track those future needed changes.